client/src/components/Home.js

```diff
@@ -1,12 +1,24 @@
 import React from "react";
+import { useHistory } from "react-router-dom";
+import { Button } from "semantic-ui-react";
 import "../styles/Home.css";
+import "semantic-ui-css/semantic.min.css";
 
 const Home = () => {
+  const history = useHistory();
+
+  function loginClick() {
+    history.push("/applicant_login");
+  }
+
   return (
     <div>
       <header className="home-header">
         <h1>Diversi-Hire</h1>
       </header>
+      <Button primary onClick={loginClick}>
+        I am an applicant
```

Comment:
If we're strictly following the acceptance criteria it should read "Log in as applicant", but as I arbitrarily chose that today, I don't think it matters much

Reply:
Ok, I can change the text. I think I can use Link instead useHistory.